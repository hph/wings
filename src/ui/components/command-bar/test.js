import React from 'react';
import renderer from 'react-test-renderer';

import { CommandBar } from './index';

const defaultProps = {
  command: '',
};

const createSnapshot = (props = defaultProps) => {
  expect(
    renderer.create(<CommandBar {...props} />).toJSON(),
  ).toMatchSnapshot();
};

test('CommandBar renders an empty command bar', () => {
  createSnapshot();
});

test('CommandBar renders a command bar with the provided command', () => {
  createSnapshot({ command: 'foo' });
});
