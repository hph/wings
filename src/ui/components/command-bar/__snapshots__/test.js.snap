// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommandBar renders a command bar with the provided command 1`] = `
<div
  className="root"
>
  <div
    className="container"
  >
    <div
      className="colon"
    >
      :
    </div>
    <div>
      foo
    </div>
    <div
      className="caret"
    />
  </div>
</div>
`;

exports[`CommandBar renders an empty command bar 1`] = `
<div
  className="root"
>
  <div
    className="container"
  >
    <div
      className="colon"
    >
      :
    </div>
    <div>
      
    </div>
    <div
      className="caret"
    />
  </div>
</div>
`;
